test/rubocop_helper.rb

```diff
@@ -0,0 +1,23 @@
+# typed: true
+# frozen_string_literal: true
+
+require "rubocop-minitest"
+require "rubocop/minitest/support"
+
+require_relative "../lib/rubocop/cop/ruby_lsp/use_language_server_aliases"
+
+module InjectRubyLspCop
+  def setup
+    super
+    return unless @cop.nil?
+
+    cop_name = self.class.to_s.delete_suffix("Test")
+    return unless ::RuboCop::Cop::RubyLsp.const_defined?(cop_name)
+
+    # rubocop:disable Sorbet/ConstantsFromStrings
+    @cop = ::RuboCop::Cop::RubyLsp.const_get(cop_name).new
```

Comment:
This is to replace https://github.com/rubocop/rubocop-minitest/blob/v0.29.0/lib/rubocop/minitest/assert_offense.rb#L82

Reply:
Instead of using this module, can't we set `@cop` in `UseLanguageServerAliasesTest` manually? For example

```ruby
class UseLanguageServerAliasesTest < Minitest::Test
  def setup
    @cop = ::RuboCop::Cop::RubyLsp::UseLanguageServerAliases.new
  end
end
```